proccess_project.py

```diff
@@ -0,0 +1,45 @@
+import multiprocessing
+import requests
+import json
+
+def parseDownload(json_object):
+    if json_object:
+        return len(json.dumps(json_object))
+    return 0
+
+def downloader(url, queue):
+    downloaded_json = requests.get(url).json()
+    url_char_num = parseDownload(downloaded_json)
+    queue.put((url, url_char_num))
+
+def main():
+    queue = multiprocessing.Queue()
+    total_chars = 0
+    tracker_dict = {}
+    urls = [
+        'https://jsonplaceholder.typicode.com/posts',
+        'https://jsonplaceholder.typicode.com/comments',
+        'https://jsonplaceholder.typicode.com/albums',
+        'https://jsonplaceholder.typicode.com/photos',
+        'https://jsonplaceholder.typicode.com/todos',
+        'https://jsonplaceholder.typicode.com/users'
+    ]
+    processes = [] 
+    for url in urls:
+        process = multiprocessing.Process(target = downloader, args = (url, queue))
+        process.start()
+        processes.append(process)
+    for _ in processes:
+        url, char_count = queue.get()
+        tracker_dict[url] = char_count
+        total_chars += char_count
+    for process in processes:
+        process.join()
```

Comment:
There is an issue here.
How come we collect data from the processes, before they finished working?
process.join waits for the process to end, but the "join" loop happens after the counting.
This might cause race condition